src/templates/csr/fabric-xl/index.svelte

```diff
@@ -5,7 +5,6 @@
 	export let TrackingPixel: GAMVariable;
 	export let ResearchPixel: GAMVariable;
 	export let ViewabilityPixel: GAMVariable;
-	export let ShowLabel: GAMVariable;
```

Comment:
Just a heads up, no action required:

This variable will still exist in the native format as this codebase just updates the native styles. Updating the format is still something that needs to be done manually, but we should wait until after this PR is merged. I don't think there's any issues keeping the format as is for now - just some tidying up for later down the line.

Reply:
Great, ill keep an eye on the deploy 🙏